src/main/resources/logback.xml

```diff
@@ -65,6 +65,12 @@
   <!-- Suppress info logs from azure packages -->
   <logger name="com.azure" level="warn" />
 
+  <!-- If you want to debug realtime updates set these loggers to debug -->
+  <logger name="org.opentripplanner.updater.stoptime.TimetableSnapshotSource" level="info" />
+  <logger name="org.opentripplanner.model.Timetable" level="warn" />
+  <logger name="org.opentripplanner.ext.siri.SiriTimetableSnapshotSource" level="warn" />
+  <logger name="org.opentripplanner.ext.siri.TimetableHelper" level="warn" />
```

Comment:
Is it intentional that these have warn as default? I don't know if it's better for these classes than info but just wanted to ask if you had left them like that by accident.